app/services/analytics.rb

```diff
@@ -76,12 +75,6 @@ def first_event_this_session?
     @session[:first_event]
   end
 
-  def register_doc_auth_step_from_analytics_event(event, attributes)
-    return unless user && user.class != AnonymousUser
```

Comment:
Is this check for `AnonymousUser` retained in the replacement code?

Reply:
it is not, but it would be *extremely* surprising if an anonymous (logged out) user was able to make it all the way through IDV, I figured it was not worth carrying over